homeassistant/components/flexmeasures/const.py

```diff
@@ -10,5 +10,6 @@
 RESOLUTION = "PT15M"
 SIGNAL_UPDATE_SCHEDULE = "update_schedule"
 FRBC_CONFIG = "frbc_config"
-SCHEDULE = "schedule"
-FLEXMEASURES_SCHEDULE = "flexmeasures_schedule"
+SCHEDULE_STATE = "schedule"
+SCHEDULE_ENTITY = "flexmeasures_schedule"
+SOC_UNIT = "MWh"
```

Comment:
For the purpose of home automation, we should get the client to accept kWh and Wh units asap.